app/V2/votes/views.py

```diff
@@ -0,0 +1,65 @@
+from datetime import datetime
+
+from flask import make_response, jsonify, Blueprint, request
+from flask_jwt_extended import jwt_required, get_jwt_identity
+
+from app.V2.auth.models import User
+from app.V2.offices.models import Office
+from app.V2.votes.models import Vote
+from utils.input_validators import Validate
+
+votes_base = Blueprint('votes-base', __name__, url_prefix='/api/v2')
+
+
+@votes_base.route('/votes', methods=['POST'])
+@jwt_required
+def post():
+    """End point to vote"""
+    data = request.get_json(force=True)
+    if len(data) < 2:
+        return make_response(jsonify({
+            "status": 400,
+            "error": "office and candidate are required"
+        }), 400)
+
+    if isinstance(data['office'], int) or isinstance(data['candidate'], int):
+        return make_response(jsonify({
```

Comment:
Hey, I would suggest using `if not isinstance(data['office'], str)` rather than `if isinstance(data['office'], int)` since with the latter you are only checking if the input is of type `int` meaning if I provide type `float` this validation won't work. 

What do you think?

Reply:
Hey.Thank you for observing that.I had not noticed.let me work on it

Reply:
> Hey, I would suggest using `if not isinstance(data['office'], str)` rather than `if isinstance(data['office'], int)` since with the latter you are only checking if the input is of type `int` meaning if I provide type `float` this validation won't work.
> 
> What do you think?

Hey @petermwash I have rectified the validation.Thanks for noticing that.